### PR TITLE
Feature/#161790868/view article

### PR DESCRIPTION
**Pull request by udochukwu**
**What does this PR do?**

- Users should be able to view a single Article

**Description of Task to be completed?**

- Setup View article store, containers, and component

**What are the relevant pivotal tracker stories?**

- https://www.pivotaltracker.com/story/show/161790868

**Screenshots (if appropriate)**

- N/A

**Questions**

- N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,18 +38,22 @@
...
     "babel-preset-react": "^6.24.1",
+    "dotenv": "^6.1.0",
     "dotenv-webpack": "^1.5.7",
     "eslint": "^5.6.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-plugin-import": "^2.14.0",
     "express": "^4.16.4",
     "jwt-decode": "^2.2.0",
     "materialize-css": "^1.0.0-rc.2",
+    "moment": "^2.22.2",
```
Comment (udochukwu) [inline package.json]: Added MomentJS for manipulating and formatting date and time

File: package.json [hunk 2]
```diff
@@ -38,18 +38,22 @@
...
     "npm-watch": "^0.4.0",
     "prop-types": "^15.6.2",
     "query-string": "^6.2.0",
     "react": "^16.6.0",
+    "react-avatar": "^3.4.6",
```
Comment (udochukwu) [inline package.json:56]: This package generates dummy profile images for users based on their username.

File: package.json [hunk 3]
```diff
@@ -38,18 +38,22 @@
...
     "react-dom": "^16.6.0",
+    "react-html-parser": "^2.0.2",
```
Comment (udochukwu) [inline package.json:58]: This package parses strings to HTML format.

**Comment by kampkelly** [comment]
LGTM

**Review by s-akinrele — APPROVED** [review]

**Pull request merged**